### PR TITLE
ISLANDORA-2048: Add PIDs to list of Simple Workflow objects

### PR DESCRIPTION
**Pull request by bondjimbond**
**JIRA Ticket**: (https://jira.duraspace.org/browse/ISLANDORA-2048)

# What does this Pull Request do?

Adds the object PID in parentheses next to object label, to make it easier to identify them and which namespace they belong to.

# What's new?

Adds the $pid variable in parentheses to the $row that displays the object. No other changes.

# How should this be tested?

- Make several objects inactive
- View the list of Simple Workflow objects
- Verify that PID displays as expected next to object label

# Interested parties
@DiegoPino, @Islandora/7-x-1-x-committers

### CODE REVIEW

**Comment by DiegoPino** [comment]
@bondjimbond busy week! Is Wednesday ok for me testing this Thanks!

**Comment by bondjimbond** [comment]
No problem at all, unless someone else wants to test first!

**Comment by bondjimbond** [comment]
Whoops, forgot to check the maintainer. Also pinging @jordandukart

**Review by DiegoPino — APPROVED** [review]
Looks like pretty straightforward! Approving. Will merge in 24 hrs if nobody opposes.

**Comment by DiegoPino** [comment]
@bondjimbond 👀

**Pull request merged**
